static/js/hotkey.js

```diff
@@ -41,6 +41,7 @@ var keydown_shift_mappings = {
     // these can be triggered by shift + key only
     9: {name: 'shift_tab', message_view_only: false}, // tab
     32: {name: 'shift_spacebar', message_view_only: true},  // space bar
+    68: {name: 'shift_d', message_view_only: false},
```

Comment:
We actually use this section only for arrow keys.   You can see plenty of examples of mapping capital letters further down in the file.

Reply:
Cool. I will fix this soon. Thanks for reviewing.

Reply:
@showell but then it will be like uppercase D restoring the last draft which can also be achieved through caps lock and then pressing d. There will be no need for shift key then. Please guide me. Thanks for you time.

Reply:
@kritisingh1 Yes! That's what we're talking about.

Reply:
@lonerz So I need to "D" key as a hotkey/shortcut key for restoring the latest draft without opening the overlay ?

Reply:
Yes. `D` is the hotkey we want.